docs/mdbook/SUMMARY.md

```diff
@@ -11,8 +11,9 @@
 
 # Platforms
 
-- [Platform-specific options](./modules/wrapper-options.md)
-- [Standalone Usage](./modules/standalone.md)
+- [Nixvim Platforms](./platforms/index.md)
+@PLATFORM_OPTIONS@
+	- [Standalone](./platforms/standalone.md)
```

Comment:
That's intentional, because the `@PLATFORM_OPTIONS@` will be replaced with a tab-indented list of links.

The final patched file will look like:
```md
- [Nixvim Platforms](./platforms/index.md)
	- [Nixos](./platforms/nixos.md)
	- [home-manager](./platforms/hm.md)
	- [nix-darwin](./platforms/darwin.md)
	- [Standalone](./platforms/standalone.md)
```

We could add a `<!-- comment -->` or switch to using space-indentation, but the former is a little clunky and the latter increases the scope of the PR, since we'd also want to update how `@NIXVIM_OPTIONS@` is indented for consistency.

Reply:
This indented list comes from here https://github.com/MattSturgeon/nixvim/blob/3da9316266e0a9a247053e64f982a29f78222668/docs/mdbook/default.nix#L254-L257

While the other one is indented here https://github.com/MattSturgeon/nixvim/blob/3da9316266e0a9a247053e64f982a29f78222668/docs/mdbook/default.nix#L221-L225

Reply:
Sorry, missed that this was the indented list.